modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/edit/EditAppBody.es.js

```diff
@@ -14,15 +14,33 @@
 
 import ClayLayout from '@clayui/layout';
 import {SearchInput} from 'data-engine-taglib';
-import React, {useState} from 'react';
+import React, {useContext, useState} from 'react';
 
 import {useRequest} from '../../../hooks/index.es';
 import {getLocalizedValue} from '../../../utils/lang.es';
+import EditAppContext, {
+	UPDATE_DATA_LAYOUT_ID,
+	UPDATE_DATA_LIST_VIEW_ID,
+	UPDATE_WORKFLOW_PROCESS_ID,
+} from './EditAppContext.es';
 import ListItems from './ListItems.es';
 
-export default ({defaultLanguageId, endpoint, title, ...restProps}) => {
+export default ({
+	defaultLanguageId,
+	endpoint,
+	itemType,
+	title,
+	...restProps
+}) => {
```

Comment:
@gabrielpramos, what do you think about export it as a named component? And rename it to `EditAppBodyStep` ?

Like this:

`export default function EditAppBodyStep({...}) `